fili-core/src/test/groovy/com/yahoo/bard/webservice/async/AsyncResultsReadySpec.groovy

```diff
@@ -0,0 +1,105 @@
+package com.yahoo.bard.webservice.async
+
+import com.yahoo.bard.webservice.async.jobs.jobrows.DefaultJobStatus
+import com.yahoo.bard.webservice.util.GroovyTestUtils
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+/**
+ * Verifies that when the results of an asynchronous request are ready, that the job status is updated to ready and
+ * the results are accessible through both the {@code results} and {@code syncResults} links.
+ */
```

Comment:
So once the results are ready, we set the result status to 'success' right? I thought the only 3 status messages we had were 'success, pending and failure'